examples/document/basic-example.html

```diff
@@ -2478,7 +2481,7 @@
   break-after: always;
 }
 
-@page :left {
```

Comment:
Could you please rebase on master, install dependencies to fetch the latest `"@asciidoctor/cli` which fix an issue with doctype, and generate the `basic-example` again?

Reply:
I can take care of it tomorrow.